render/detailed_node_test.go

```diff
@@ -109,6 +111,6 @@ func TestMakeDetailedNode(t *testing.T) {
 		},
 	}
 	if !reflect.DeepEqual(want, have) {
-		t.Errorf("%s", diff(want, have))
+		t.Errorf("%s", test.Diff(want, have))
```

Comment:
Yeah me neither, but a whole package for a 20 line file seems overkill.  At least with the name test, it might one day contain other things.